docs/manual_doc/Statement_Syntax/Data_Definition_Statements/Drop_Space_Syntax.md

```diff
@@ -0,0 +1,10 @@
+```
+DROP SPACE space_name
+```
+You must have the DROP privilege for the graph space.
+DROP SPACE deletes everyting (all the vertices, edges, indices, and properties) in the specific space.  
+Other spaces remain unchanged.
+This statement does not immediately remove all the files and directories that holds in the storage engine (and release disk space). The deletetion depends on the implement of  different storage engine.
```

Comment:
This statement does not immediately remove all the files and directories in the storage engine (and release disk space).